sensor_msgs/CompressedImage image
string[] fast_sam_strings
---
sensor_msgs/Image segmentation
sensor_msgs/Image traversable_segmentation
int32[] mask_sums
int32[] coordinates
uint32[] segmentation_shape
uint32[] non_zeros_indices_shape
uint32[] non_zeros_indices